relayer/headers.go

```diff
@@ -100,6 +100,16 @@ func (uh *SyncHeaders) GetTrustedHeaders(src, dst *Chain) (srcTh, dstTh *tmclien
 	return
 }
 
+func (uh *SyncHeaders) GetTrusteddsthHeader(src, dst *Chain) (dstTh *tmclient.Header, err error) {
```

Comment:
Maybe rename to `GetTrustedHeader`, but construct the call like:

`sh.GetTrsutedHeader(dst, c)`

note the swapping. This way we can reuse this function for source if needed